src/pages/workspace/companyCards/WorkspaceCompanyCardAccountSelectCardPage.tsx

```diff
@@ -42,7 +42,8 @@ function WorkspaceCompanyCardAccountSelectCardPage({route}: WorkspaceCompanyCard
     const [allBankCards] = useCardsList(policyID, bank as CompanyCardFeed);
     const card = allBankCards?.[cardID];
     const connectedIntegration = getConnectedIntegration(policy) ?? CONST.POLICY.CONNECTIONS.NAME.QBO;
-    const exportMenuItem = getExportMenuItem(connectedIntegration, policyID, translate, policy, card, Navigation.getActiveRoute());
+    const activeRoute = useMemo(() => Navigation.getActiveRoute(), []);
```

Comment:
NAB: Should we add a comment explaining why `Navigation.getActiveRoute()` is in a `useMemo` so someone doesn't remove it by mistake?